src/ui.js

```diff
@@ -153,7 +153,25 @@ const executeQueryFromTerminalUI = async (queryOptions, successCb, errorCb)  =>
     headers,
   } = queryOptions;
   cli.action.start('Introspecting schema');
-  const schemaResponse = await query({endpoint: endpoint, query: introspectionQuery, headers: headers});
+
+  let schemaResponse;
+  try {
+    schemaResponse = await query({endpoint: endpoint, query: introspectionQuery, headers: headers});
+  } catch (err) {
+    if (err.message && err.message.startsWith('Network error: Unexpected token')) {
```

Comment:
@blackdahila you might want to check this condition. I tried running the command:

```sh
./bin/run http://localhost:random --introspect
```

It gave me this error:

```js
Executing query... error
Error:  ApolloError: Network error: request to http://localhost/:random failed, reason: connect ECONNREFUSED 127.0.0.1:80
    at new ApolloError (/home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:124:32)
    at /home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:1248:45
    at /home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:1680:21
    at Array.forEach (<anonymous>)
    at /home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:1679:22
    at Map.forEach (<anonymous>)
    at QueryManager.broadcastQueries (/home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:1672:26)
    at /home/wawhal/oss/graphql/graphqurl/node_modules/apollo-client/bundle.umd.js:1175:35
    at processTicksAndRejections (internal/process/task_queues.js:85:5) {
  graphQLErrors: [],
  networkError: FetchError: request to http://localhost/:random failed, reason: connect ECONNREFUSED 127.0.0.1:80
      at ClientRequest.<anonymous> (/home/wawhal/oss/graphql/graphqurl/node_modules/node-fetch/lib/index.js:1345:11)
      at ClientRequest.emit (events.js:203:13)
      at Socket.socketErrorListener (_http_client.js:399:9)
      at Socket.emit (events.js:203:13)
      at emitErrorNT (internal/streams/destroy.js:91:8)
      at emitErrorAndCloseNT (internal/streams/destroy.js:59:3)
      at processTicksAndRejections (internal/process/task_queues.js:77:11) {
    message: 'request to http://localhost/:random failed, reason: connect ECONNREFUSED 127.0.0.1:80',
    type: 'system',
    errno: 'ECONNREFUSED',
    code: 'ECONNREFUSED'
  },
  message: 'Network error: request to http://localhost/:random failed, reason: connect ECONNREFUSED 127.0.0.1:80',
  extraInfo: undefined
}
```